### PR TITLE
Update past pages

### PR DESCRIPTION
**Pull request by chris-gds**
## What 

Update [Past Prime Minster's page](https://www.gov.uk/government/history/past-prime-ministers), [Past Chancellor page](https://www.gov.uk/government/history/past-chancellors) & [Past Foreign Secretaries Page](https://www.gov.uk/government/history/past-foreign-secretaries) to apply new Design + Design system + use Components

## Why 

On-going work to align architecture, have pages benefit from Accessibility improvements

### Updated Combined PR of:
https://github.com/alphagov/whitehall/pull/6236 
https://github.com/alphagov/whitehall/pull/6243
https://github.com/alphagov/whitehall/pull/6241
(all converted to draft - will close)

## Visuals 

| Before      | After* |
| ----------- | ----------- |
| www gov uk_government_history_past-prime-ministers(iPhone 6_7_8 Plus) | 1www gov uk_government_history_past-prime-ministers(iPhone-6_7_8-Plus) |
|  |
| www gov uk_government_history_past-prime-ministers (1)  | screenshot-www integration publishing service gov uk-2021 08 31-12_55_10 |

| Before      | After |
| ----------- | ----------- |
| www gov uk_government_history_past-chancellors(iPhone 6_7_8) | mobile-after |
| ----------- | ----------- |
| www gov uk_government_history_past-chancellors | desktop-after |

| Before | After |
| ----------- | ----------- |
| www gov uk_government_history_past-foreign-secretaries(iPhone-6_7_8-Plus) | www gov uk_government_history_past-foreign-secretaries(Moto G4) |
|  |
| www gov uk_government_history_past-foreign-secretaries | www gov uk_government_history_past-foreign-secretaries (1) | 

## Anything else?

To prevent missing Content for  "`theresa-may`" blocking this PR this link has been removed - hopefully temporarily 
(note: screenshots still have link present)

* `image_card` was extended to factor in a [new Design](https://github.com/alphagov/govuk_publishing_components/pull/2286) and 
* [lazy loading](https://github.com/alphagov/govuk_publishing_components/pull/2270) was added to the `image_card` as this maintains existing performance benefits.
* [Add link](https://github.com/alphagov/govuk_publishing_components/pull/2277) to image card heading - this is not dependant

[Discussion around Design System layout + lists can be found here](https://github.com/alphagov/whitehall/pull/6236#discussion_r695518441), [an issue capturing the concerns can be found here](https://github.com/alphagov/govuk-frontend/issues/2328)

[Design Ref](https://www.figma.com/file/UjCeRTxle1bLQv3VFNz9Wt/Redesign-of-history-pages)

### CODE REVIEW

**Review by maxgds — APPROVED** [review]
I think there is more we could do to consolidate this and reuse the same partials across the different pages, but we have achieved our aim of updating the style, layout and improving componentisation at this point, supporting our aim of a more accessible set of pages, so we should move on to improving further pages. I've raised an issue against it here: https://github.com/alphagov/whitehall/issues/6265

Consider squashing some of the commits together so the merge is cleaner.

File: app/views/historic_appointments/_role_appointment.html.erb [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-  <%= render "govuk_publishing_components/components/image_card", {
-    href: historic_appointment_path(role.historic_param, role_appointment.person),
-    image_src: role_appointment.person.image_url(:s216),
-    image_loading: "lazy",
-    heading_text: role_appointment.person.name,
-    extra_links: previous_dates_in_office_prime(role, role_appointment.person, political_membership),
-    extra_links_no_indent: true
-  } %>
+  <%= content_tag_for :div, role_appointment, class: 'govuk-grid-column-one-quarter', role: "listitem" do %>
+    <% if historic_appointment_path(role.historic_param, role_appointment.person) != "/government/history/past-prime-ministers/theresa-may" %>
```
Comment (maxgds) [inline app/views/historic_appointments/_role_appointment.html.erb:3]: on the off chance that this problem hangs around for a long time, please add a comment to the code explaining why we've had to do this.

Comment (chris-gds) [reply app/views/historic_appointments/_role_appointment.html.erb:3]: Will do, I'll raise an issue as well linking it

Comment (chris-gds) [reply app/views/historic_appointments/_role_appointment.html.erb:4]: REF: https://github.com/alphagov/whitehall/issues/6266

**Comment by chris-gds** [comment]
>Consider squashing some of the commits together so the merge is cleaner.

This has been done, could you merge once happy @maxgds

**Pull request merged**